libs/sdk-ui-charts/src/highcharts/adapter/plugins/adjustTickAmount.ts

```diff
@@ -47,7 +47,7 @@ function isYAxis(axis: Highcharts.Axis): boolean {
  * @returns true if any axis is set min/max to. Otherwise false
  */
 function isUserSetExtremesOnAnyAxis(chart: Highcharts.Chart): boolean {
-    const yAxes = chart.userOptions.yAxis;
+    const yAxes: any = chart.userOptions.yAxis;
```

Comment:
I do not understand how is possible that previous code is working

Remove any and check it for array 

const yAxis = chart?.userOptions?.yAxis;

    if (yAxis && isArray(yAxis)) {
        return (yAxis[0]?. isUserMinMax) || (yAxis[1]?. isUserMinMax);
    }

Reply:
I added the array check, but we cannot fully omit the any as isUserMinMax looks like a custom property which is not typed anywhere

Reply:
ok